cmd/av/stack_branch.go

```diff
@@ -13,6 +14,8 @@ var stackBranchFlags struct {
 	// The parent branch to base the new branch off.
 	// By default, this is the current branch.
 	Parent string
+	// If true, move/rename the current branch.
+	Move bool
```

Comment:
I'm not sure this is self explanatory. If we are doing renaming, we should just call it `Rename`